lmnet/lmnet/utils/predict_output/output.py

```diff
@@ -16,15 +16,14 @@
 import base64
 import json
 import os
-from datetime import datetime, timezone
+from datetime import datetime
 from io import BytesIO
 
 import numpy as np
 import PIL.Image
 import PIL.ImageDraw
-from matplotlib import cm
```

Comment:
can we remove `matplotlib` from `requirements.txt`?

Reply:
After this PR, only `lmnet/networks/classification/base.py` will depend on `matplotlib`.
https://github.com/blue-oil/blueoil/blob/8055a4a29618d79cf70d068d5aac5af7a4f3351e/lmnet/lmnet/networks/classification/base.py#L17
Is it worth adding a new dependency for visualization? If not, I would remove it in another PR and reimplement this.
Related to: https://github.com/blue-oil/blueoil/issues/485

Reply:
I see. I think it'd be nice.